cloudevents/SDK.md

```diff
@@ -109,9 +109,16 @@ based on the parameters set, most importantly the CloudEvents spec version.
 
 #### Encode/Decode an Event
 
-Each SDK will support encoding and decoding an Event with regards to a transport
-and encoding. `Structured` encoding is the easiest to support, as it is just
-`json`, but `Binary` is fairly custom for each transport.
+Each SDK MUST support encoding and decoding an Event with regards to a transport
+and encoding:
+
+- Each SDK MUST support structured-mode messages for each transport that it
+  supports.
+- Each SDK SHOULD support binary-mode messages for each transport that it
+  supports.
+- Each SDK SHOULD support batch-mode messages for each transport that it
+  supports (where the event format and transport combination supports batch mode).
+- Each SDK MUST indicate which modes it supports for each supported event format.
```

Comment:
We should probably say how it needs to do this... docs vs the support table we just added...

Reply:
Let's discuss what we want to require in the meeting.

Reply:
Okay:

- Both SDK-specific docs and the support table (including a link)
- Change to SHOULD

Reply:
Done in the next commit.